pagerduty/plugin.go

```diff
@@ -33,6 +33,7 @@ func Plugin(ctx context.Context) *plugin.Plugin {
 			"pagerduty_team":                tablePagerDutyTeam(ctx),
 			"pagerduty_user":                tablePagerDutyUser(ctx),
 			"pagerduty_vendor":              tablePagerDutyVendor(ctx),
+			"pagerduty_oncalls":             tablePagerDutyOnCalls(ctx),
```

Comment:
Could you please sort the table names in ascending order?

Reply:
We keep our table names in the singular. Also On Call are two separate words in Pagerduty.